assets/scene_renderer/shaders/DebugMaterial.hlsl

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#define ENABLE_VTX_ATTR_TEXCOORD
+#define ENABLE_VTX_ATTR_NORMAL
+#define ENABLE_VTX_ATTR_TANGENT
+
+#include "MaterialInterface.hlsli"
+
+#define VTX_POSITION 0
+#define VTX_TEXCOORD 1
+#define VTX_NORMAL   2
+#define VTX_TANGENT  3
```

Comment:
i don't see this field in that file? (please see materialinterface.hlsli in this PR)

Reply:
`dbgVtxAttrIndex` is defined in `materialinterface.hlsli`
Since those defined seem to only be used to set the magic number inside this field, maybe they should be written close to it. (So if multiple shaders use this field, they can agree on the magic numbers definitions)

Reply:
ohhh - i see what you mean. yeah, i agree; i'll move this